packages/amazon-cognito-identity-js/tsconfig.build.json

```
@@ -0,0 +1,5 @@
+{
+	"extends": "../tsconfig.settings.json",
```

Very nit: this is a more common name.
```suggestion
	"extends": "../tsconfig.base.json",
```

---

done